WEB-INF/tags/eventForm.tag

```diff
@@ -0,0 +1,39 @@
+      <%-- Form --%>
+      <form action="${pageContext.request.contextPath}/maintenance" method="POST" accept-charset="UTF-8">
+              <input name="utf8" value="?" type="hidden" />
```

Comment:
Something is very weird about this indentation... the first line is indented 6 spaces, then 7, and in some places 2. Could you press `ctrl+alt+l` for me and see if that fixes it? If not, let me know here

Reply:
Also, please set the value on that hidden input to `✓`. You can copy and paste it!

Reply:
Hey Sean. I've pressed ctrl+alt+l and it said that my lines were all properly formatted. So must've fucked up my settings along the lines. Any suggestions let me know, I'll look into it now :)